.github/workflows/publish.yml

```diff
@@ -82,6 +82,8 @@ jobs:
           tar -xvf artifact.tar
           npm install linkinator
           npx linkinator . --recurse --markdown \
+            --skip https://groups.google.com/* \
+            --skip https://github.com/ministryofjustice/dns \
```

Comment:
duplication

Reply:
I don't remember putting that there, but will remove.